src/Console/CacheItemPoolFactoryInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Contentful\Console;
+
+use Psr\Cache\CacheItemPoolInterface;
+
+interface CacheItemPoolFactoryInterface
+{
+    public function __construct();
```

Comment:
I think we should pass `$spaceId` as parameter here. Some cache implementations could use that for namespacing the cache items, so it's best to at least provide that option.

Reply:
Noted, but I since I think it would be better to pass that in the `getCacheItemPool()` method in that case, I'll put it there for now :)